addons/skin.estuary/xml/DialogMusicInfo.xml

```diff
@@ -444,39 +469,39 @@
 						<top>480</top>
 						<control type="list" id="41">
 							<left>42</left>
-							<top>0</top>
+							<top>9</top>
```

Comment:
Further adjustment needed to info panel, as focus texture does not align to the top of the panel. It was right before but this has made it worse.

**Before** - small gap between top of panel and first item in focus.
![image](https://github.com/user-attachments/assets/2a9f1315-31c5-4c6f-ad33-410ac53eed3c)

**After** - larger gap between top of panel and first item in focus.

![image](https://github.com/user-attachments/assets/4c89627f-fb85-4e5e-92c3-404e981a21e6)

In contrast the video info dialog panel the list aligns to top of panel so there no gap between top of panel and the first item in focus. So in my view we should aim to do the same in the music info dialog.
![image](https://github.com/user-attachments/assets/7e6d4fa7-676c-4161-aa9c-492fad682252)

Reply:
I see the video info dialog uses a grouplist instead of a list and suffers from the same overlapping problem I found in the bottom row of buttons.

![screenshot00005](https://github.com/user-attachments/assets/ba847468-94a6-4f86-ac93-305b6aae76b7)

More work is needed.